src/Tickets/Seating/Orders/Controller.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * The controller for the Seating Orders.
+ *
+ * @since TBD
+ *
+ * @package TEC/Tickets/Seating/Orders
+ */
+
+namespace TEC\Tickets\Seating\Orders;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\lucatume\DI52\Container;
+use TEC\Tickets\Commerce\Status\Status_Interface;
+use WP_Post;
+use Tribe__Tickets__Tickets;
+
+/**
+ * Class Controller
+ *
+ * @since TBD
+ *
+ * @package TEC/Tickets/Seating/Orders
+ */
+class Controller extends Controller_Contract {
+	
+	/**
+	 * A reference to Attendee data handler
+	 *
+	 * @since TBD
+	 *
+	 * @var Attendee
+	 */
+	private Attendee $attendee;
+	
+	/**
+	 * A reference to Cart data handler
+	 *
+	 * @since TBD
+	 *
+	 * @var Cart
+	 */
+	private Cart $cart;
+	
+	/**
+	 * Controller constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param Container $container The DI container.
+	 * @param Attendee  $attendee   The Attendee data handler.
+	 * @param Cart      $cart       The Cart data handler.
+	 */
+	public function __construct( Container $container, Attendee $attendee, Cart $cart ) {
+		parent::__construct( $container );
+		$this->attendee = $attendee;
+		$this->cart     = $cart;
+	}
+	
+	/**
+	 * The action that will be fired when this Controller registers.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	protected function do_register(): void {
+		// Todo: This is only handling seating info for TicketsCommerce for now, in future we have to include other actions for other providers.
```

Comment:
Other commerce providers will be in ET+, though, right? I think this TODO can be removed from here.